openpype/lib/avalon_context.py

```diff
@@ -494,6 +497,18 @@ def get_workfile_template_key(
     return default
 
 
+def get_system_general_data():
```

Comment:
You've just created this function which is used only in this file and not used at any other place so why it is separated to function?

Reply:
I have separated it to function as it is used on 3 places in this modul. But yeah it can be nonpublic.